src/app/modules/category/components/filter/filter.component.html

```diff
@@ -1 +1,102 @@
-<div class="filter"></div>
+<div class="filter" *ngIf="isReadyToDisplay$ | async; else loading">
+    <div class="filter-wrap">
+        <div
+            class="filter-products"
+            *ngFor="let items of selectFilterData$ | async; let i = index"
+        >
+            <h4 class="filter-products-title">{{ items.filter_title }}</h4>
+            <section class="checkbox-section" *ngIf="items.type === 0">
+                <ul class="filter-products-list">
+                    <li class="item" *ngFor="let item of items.filters">
+                        <mat-checkbox
+                            value="{{ item }}"
+                            (change)="onChecked(item, $event.checked, i)"
+                            >{{ item }}
+                        </mat-checkbox>
+                    </li>
+                </ul>
+            </section>
+            <section class="inputs-section" *ngIf="items.type === 1">
+                <ul class="filter-products-list">
+                    <li class="item">
+                        <div class="inputs-wrap">
+                            <input
+                                class="input"
+                                [(ngModel)]="inputValue1"
```

Comment:
u bind multiple inputs to 1 class variable, but we should use n independent values for n inputs